iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/PhaseTapChangerImpl.java

```diff
@@ -83,17 +87,23 @@ public double getRegulationValue() {
 
     @Override
     public PhaseTapChangerImpl setRegulationValue(double regulationValue) {
-        ValidationUtil.checkPhaseTapChangerRegulation(parent, regulationMode, regulationValue, isRegulating(), getRegulationTerminal(), getNetwork());
         int variantIndex = network.get().getVariantIndex();
+        ValidationUtil.checkCurrentOrActivePowerSetpoint(parent, VALIDABLE_TYPE_DESCRIPTION, regulationValue, isRegulating());
         double oldValue = this.regulationValue.set(variantIndex, regulationValue);
         String variantId = network.get().getVariantManager().getVariantId(variantIndex);
         notifyUpdate(() -> getTapChangerAttribute() + ".regulationValue", variantId, oldValue, regulationValue);
         return this;
     }
 
+    @Override
+    public PhaseTapChangerImpl setTargetDeadband(double targetDeadband) {
+        ValidationUtil.checkTargetDeadband(parent, VALIDABLE_TYPE_DESCRIPTION, targetDeadband, isRegulating());
```

Comment:
If you don't delete the check in the parent method, this one is not useful

Reply:
explained above